utils/cSeedMapGenerator.h

```diff
@@ -37,38 +37,38 @@ class cSeedMapGenerator {
 
 	/* these is array used to find corresponding points */
 	/*
-	      A�B�C    00��20��40
-	      �����    ����������
-	      D�E�F    02��22��42
-	      �����    ����������
-	      G�H�I    04��24��44
+	      A�B�C    00��20��40
```

Comment:
what happened here? :)

Reply:
Oh, I didn't notice that! Let's not merge this as-is then.
I did see some messages on line breaks and carriage returns being converted from windows style to unix style, so perhaps that's where it went wrong...

Reply:
The characters were unicode dots (<B7>), not ASCII dots. My VSCode messed them up, I think. I've replaced them with ASCII dots.